var/spack/repos/builtin/packages/pgplot/package.py

```diff
@@ -0,0 +1,68 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Pgplot(MakefilePackage):
+    """PGPLOT Graphics Subroutine Library
+
+    The PGPLOT Graphics Subroutine Library is a Fortran- or
+    C-callable, device-independent graphics package for making
+    simple scientific graphs. It is intended for making
+    graphical images of publication quality with minimum effort
+    on the part of the user. For most applications, the program
+    can be device-independent, and the output can be directed to
+    the appropriate device at run time."""
+
+    homepage = "https://sites.astro.caltech.edu/~tjp/pgplot/"
+    url      = "ftp://ftp.astro.caltech.edu/pub/pgplot/pgplot5.2.tar.gz"
+
+    maintainers = ['eschnett']
+
+    version('5.2.2',
+            url="ftp://ftp.astro.caltech.edu/pub/pgplot/pgplot5.2.tar.gz",
+            sha256='a5799ff719a510d84d26df4ae7409ae61fe66477e3f1e8820422a9a4727a5be4')
+
+    patch('conf')
```

Comment:
Can you add a comment similar to the one requested above also for this patch directive?

Reply:
Done.

Reply:
Both Lorene and pgplot are configured by copying/modifying files that define make variables, specifying compilers, paths, etc. I point to Spack's wrappers.

I use a patch to make this happen because this seemed a convenient way to create a small file. Alternatives would be possible, e.g. creating the files from a Python string.